Packs/Automox/pack_metadata.json

```
@@ -1,19 +1,45 @@
 {
     "name": "Automox",
-    "description": "Automox Content Pack for Demisto",
+    "description": "This pack empowers you with comprehensive solutions to manage your Automox device fleet with ease!",
     "support": "partner",
     "currentVersion": "1.0.0",
     "author": "Automox Inc.",
     "url": "https://www.automox.com/",
     "email": "support@automox.com",
     "categories": [
-        "Endpoint"
+        "Endpoint",
+        "Vulnerability Management"
+    ],
+    "tags": [
+        "IT",
+        "Network",
+        "Security",
+        "Use Case",
+        "New"
+    ],
+    "useCases": [
+        "IT Services",
+        "Vulnerability Management"
+    ],
+    "keywords": [
+        "Automox",
+        "Patch",
+        "Endpoint",
+        "Vulnerability",
+        "CVE",
+        "Vulnerability Sync"
```

```suggestion
        "CVE"
```
lets scratch vulnerability sync for now